src/components/util/split-button.js

```diff
@@ -68,8 +74,10 @@ const SplitButton = (props) => {
     };
 
     const handleMenuItemClick = (event, index) => {
-        if (props.onSelectionChange) {
+        if (!props.isRunning) {
```

Comment:
why remove the check for onSelectionChange ?

Reply:
property onSelectionChange is always defined. I will add required on this property